### DISCUSSION TITLE
Tag filtering

### DISCUSSION THREAD

**Discussion by JPRB** [dailydotdev/daily#1082] [Feature Requests]
Instead of having all the information in the feed, give the option of having blocks by tag.

For example, if the user subscribed to the Cloud tag, they would be able to see information only about that tag, instead of everything being mixed together.

Another option is, in addition to the main Tag (Cloud, Databases, DevOps, ...), the information is in blocks divided by sub-tags.
For example, in the case of Tag Databases, divide into each of the tags: Sql, redis, postgres.

**Comment by idoshamun** [comment]
Hi @JPRB,

That's a great idea. We already have active discussion here #1093
